Dockerfile

```diff
@@ -1,38 +1,45 @@
-
-
-FROM ubuntu:21.04 as build
+FROM ubuntu:20.04 as build
 
 ENV LANG C.UTF-8
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN dpkg --add-architecture i386 && \
-    apt-get update && \
+RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y \
-        binutils:i386 \
-        gcc-10:i386 \
-        g++-10:i386 \
-        python3.10 \
+        binutils \
+        gcc-10 \
+        g++-10 \
+		p7zip-full \
+        python3 \
         python \
         make \
         cmake \
+        curl \
         git \
         lld \
-        libsdl2-dev:i386 \
-        zlib1g-dev:i386 \
-        libbz2-dev:i386 \
-        libpng-dev:i386 \
-        libgles2-mesa-dev && \
-    ln -sf /usr/bin/python3.10 /usr/bin/python3 && \
-    ln -s /usr/bin/gcc-10 /usr/bin/gcc && \
-    ln -s /usr/bin/gcc-10 /usr/bin/cc && \
-    ln -s /usr/bin/g++-10 /usr/bin/g++ && \
-    ln -s /usr/bin/g++-10 /usr/bin/c++
+        libsdl2-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        libpng-dev \
+        libgles2-mesa-dev && \    
+	update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 10 && \
+	update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++10 10
+	
+RUN apt-get clean autoclean && apt-get autoremove --yes && rm -rf /var/lib/apt /var/lib/cache /var/lib/log
 
 RUN git clone https://github.com/Perlmint/glew-cmake.git && \
     cmake glew-cmake && \
     make -j$(nproc) && \
     make install ARCH64=false
-
+    
+ENV SDL2VER=2.0.22
+RUN curl -sLO https://libsdl.org/release/SDL2-${SDL2VER}.tar.gz && \
+    tar -xzf SDL2-${SDL2VER}.tar.gz && \
+    cd SDL2-${SDL2VER} && \
+    ./configure --build=i386-linux-gnu --prefix=/usr && \
```

Comment:
the i386 versions of libraries are not used any more, shouldn't this be building the x86_64 version of libsdl?

Reply:
also, you're pulling ubuntu's libsdl from the repos, maybe this step is just not needed any more (haven't verified myself).

Reply:
It's needed for the supporting files. omitting those files, leades to a large number of required files to be pulled in separately. This allows for dependencies, then overrites the libs with the updated version. Updated to correct path.